Shared/Flows/Onboarding/Welcome/WelcomeCollectionView.swift

```diff
@@ -18,7 +18,7 @@ class WelcomeCollectionView: CollectionView {
     init() {
         let layout = MessagesTimeMachineCollectionViewLayout()
         layout.messageContentState = .collapsed
-        layout.uiState = .write
```

Comment:
I cleaned up the layout object so it doesn't know anything about the different UI states. More on this later...